gravitee-apim-console-webui/src/components/gio-connector-dialog/gio-connector-dialog.component.html

```diff
@@ -0,0 +1,34 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="gio-connector-dialog__title">
+  <h2 mat-dialog-title class="title">{{ name }}</h2>
+  <mat-icon class="gio-connector-dialog__title__cancel"></mat-icon>
+  <button mat-icon-button [attr.aria-label]="'Close'" (click)="onClose()">
+    <mat-icon class="gio-connector-dialog__title__cancel" svgIcon="gio:cancel"> </mat-icon>
+  </button>
+</div>
+
+<mat-dialog-content class="gio-connector-dialog__content">
+  <div class="gio-connector-dialog__content__description">{{ pluginMoreInformation.description }}</div>
```

Comment:
maybe we should condition in case the data is not filled in? and display a default message if the description is empty?